OpenRA.Game/FileFormats/WavLoader.cs

```diff
@@ -35,23 +35,20 @@ public enum WaveType { Pcm = 0x1, ImaAdpcm = 0x11 }
 
 		bool LoadSound(Stream s)
 		{
-			var type = s.ReadASCII(4);
```

Comment:
Using the same function avoids having to parse the file twice.  These `return false`s indicate that it is not a wave file and that the public `TryParseSound` should also return false.